src/main/resources/db/dataset/init-demo-data.sql

```diff
@@ -491,20 +508,24 @@ INSERT INTO public.person (id, email, first_name, last_name, birthdate, title, p
     (56, 'test@test.com', 'Catherine',   'Renault',     898134646200, 0, TRUE,  'SABIANE14', false, NULL),
     (57, 'test@test.com', 'Jacques',     'Faure',       907844676970, 0, TRUE,  'SABIANE15', false, NULL),
     (58, 'test@test.com', 'Marion',      'Cotille',     917554707740, 1, FALSE, 'PROTO01', false, NULL),
-    (59, 'test@test.com', 'Arthur',      'Couyer',      927264738510, 1, TRUE, 'PROTO03', false, NULL),
-    (60, 'test@test.com', 'Kaa',         'Melott',      936974769280, 0, TRUE, 'PROTO05', false, NULL),
-    (61, 'test@test.com', 'Père',        'Seval',       946684800050, 0, TRUE, 'PROTO06',false, NULL),
+    (59, 'test@test.com', 'Arthur',      'Couyer',      927264738510, 1, TRUE,  'PROTO03', false, NULL),
+    (60, 'test@test.com', 'Kaa',         'Melott',      936974769280, 0, TRUE,  'PROTO05', false, NULL),
+    (61, 'test@test.com', 'Père',        'Seval',       946684800050, 0, TRUE,  'PROTO06',false, NULL),
     (62, 'test@test.com', 'Fu',          'Ture',        null,         0, TRUE,  '11', false, 'NEXT'),
-    (63, 'test@test.com', 'Opre',        'Vious',       315532800000, 0, TRUE,  '11', true, 'PREVIOUS'),
-    (64, 'test@test.com', 'Agrippa',     'Nel',         1024815788000, 0, FALSE,  '11', true, 'PREVIOUS'),
-    (65, 'test@test.com', 'Isidore',     'Champ',       1070870588000, 0, FALSE,  '11', false, 'PREVIOUS'),
-    (66, 'test@test.com', 'Bowie',       'Ken',         null,          0, FALSE,  '20', false, 'NEXT');
-
+    (63, 'test@test.com', 'Opre',        'Vious',       315532800000, 0, TRUE,  '11', true,  'PREVIOUS'),
+    (64, 'test@test.com', 'Agrippa',     'Nel',         102481578800, 0, FALSE, '11', true,  'PREVIOUS'),
+    (65, 'test@test.com', 'Isidore',     'Champ',       107087058800, 0, FALSE, '11', false, 'PREVIOUS'),
+    (66, 'test@test.com', 'Earl',      'Simmons',       383503015391,  0, FALSE,'24', TRUE,   NULL),
```

Comment:
when adding surveyunbit_24 c_h you removed Ken Bowie from survey-unit_11 previous c_h => why ?